pkg/component/runtime/service.go

```diff
@@ -275,6 +281,26 @@ func (s *serviceRuntime) Run(ctx context.Context, comm Communicator) (err error)
 	}
 }
 
+// .eaci.sock == Elastic Agent Connection Info socket
+const elasticAgentConnInfoSocket = ".eaci.sock"
+
+func getConnInfoServerAddress(os string, isLocal bool, port int) string {
+	if isLocal {
+		u := url.URL{}
+		u.Path = "/"
+
+		if os == "windows" {
+			u.Scheme = "npipe"
+			return u.JoinPath("/", elasticAgentConnInfoSocket).String()
+		}
+
+		u.Scheme = "unix"
+		return u.JoinPath(paths.InstallPath(paths.DefaultBasePath), elasticAgentConnInfoSocket).String()
```

Comment:
This is not unique from a standpoint of the Elastic Agent in the future supporting another service type. The `.eaci.sock` should be set in the component specification, not something that should be hard-coded into the Elastic Agent.

Reply:
Makes sense, will update